bigquery/snippets/job/bigquery_cancel_job.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package job
+
+// [START bigquery_cancel_job]
+
```

Comment:
I don't think `gofmt` needs the newline here, but it shows up when you include the sample. So, you may want to delete it.

Reply:
killed whitespace between start tag and start of imports in all samples.